### ISSUE TITLE
Inconvenient library

### ISSUE THREAD

**Issue by gazedash** [react-native-community/react-native-picker#128]
When using this library on both platforms, it is hard to achieve the same style or at least behavior.

If we go with inline picker, android will be pain in the ass to catch up (is it even possible to make it be NOT a modal?)
If we go with "modal" picker, we have to make special logic to handle iOS
If we just want to display this component regardless, with just a tap on our custom button? I don't know how to do it...

**Comment by cristianoccazinsp** [comment]
I agree that the iOS version could use some love. Currently, height styling and some other props are not even working, it always looks awful. 

It would be great to at least have a way to easily make the iOS version appear as a modal when pressed (just like Android).

This is what I get on iOS just using the demo/example code...

<img width="395" alt="Screen Shot 2020-11-11 at 11 00 11" src="https://user-images.githubusercontent.com/48869228/98820621-280c4680-240d-11eb-9898-fc69b47d7c61.png">
<img width="404" alt="Screen Shot 2020-11-11 at 11 00 28" src="https://user-images.githubusercontent.com/48869228/98820625-293d7380-240d-11eb-9f8a-3d3b11fb4654.png">

Height is ignored completely. Text ends up chopped on the left. I don't know if this is because I rendered it on a modal, but was just giving it a try and got awful results.

**Comment by jackdewhurst** [comment]
This library is awful. 
- Inconsistency between platforms
- Android selects the first row automatically on render for some devices
- Picker styles don't work properly on Android
- iOS requires you to roll your own UI for anything pre iOS 14

**Comment by jackdewhurst** [comment]
Another issue, Dark mode styling on iOS is not exposed.

**Comment by KrisLau** [comment]
This package be better for your use cases: https://github.com/hossein-zare/react-native-dropdown-picker
